.github/workflows/ci.yaml

```diff
@@ -0,0 +1,117 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
```

Comment:
I don't think we even need a matrix for a single job here. You can write `7.4` directly in the `setup-php` step (and remove it from the job name, as there is no point specifying a php version to validate the composer.json)

Reply:
I use it twice, in the name of the job and the job itself. I changed the name and do not name the version there and removed the matrix entirely.